add-domain

```diff
@@ -468,6 +468,7 @@ survivecovid19.ga
 survivecovid19.ml
 survivecovid19.tk
 survivvruscorona.us
+tabitha-cambodia.org//app/Model/CpgredesPerts/cgi_b!n/cg%c4%af_bin/ijj%c5%939p0ol932/871%c3%actip9oling/P%c4%abBalyc0%c5%93uk/80%c3%bce2101b%c4%8d/Bbbw%c4%99biay/lopsdjjjh%c3%aab%c3%b8jhxhas%c5%93jhghbvssiu8e90j/P%c3%bfBay/?fbclid=IwAR1F2ugJu5vjikpoyymFv4UfN0tX-mziKJGg2Mo-Up_2Q-dmGcVSJM6X0Kg
```

Comment:
Looks more like a URI than a domain :smirk: :wink:

Reply:
Ah whoops.  I guess chop off the back after the .org.  The site has been a host of two phishing links so I think it maybe good to blacklist the entire domain.